temporalio/worker/_activity.py

```diff
@@ -490,6 +490,18 @@ async def _run_activity(
                         temporalio.exceptions.CancelledError("Cancelled"),
                         completion.result.cancelled.failure,
                     )
+                elif (
+                    isinstance(
+                        err,
+                        temporalio.exceptions.ApplicationError,
+                    )
+                    and err.category
+                    == temporalio.exceptions.ApplicationErrorCategory.BENIGN
+                ):
+                    # Do not log for ApplicationError with BENIGN category.
```

Comment:
Do we still want to log at a debug level like you did at https://github.com/temporalio/sdk-go/pull/1925?

Reply:
Yes - this was a late night brain fart